src/components/About/Overview/index.js

```diff
@@ -555,382 +555,365 @@ const buttonAttributes = [
   { label: 'Math', icon: <PlusOne />, video: MathDemo },
 ];
 
-class Overview extends Component {
-  state = {
-    gifIndex: 0,
-    isVideoModalOpen: false,
-  };
+const Overview = () => {
+  const [gifIndex, setgifIndex] = useState(0);
+  const [isVideoModalOpen, setVideoModalOpen] = useState(false);
+  var exampleTime;
```

Comment:
why `var`?

Reply:
Just because of `function-level` scoping.

Reply:
Please use ES6 syntax for consistent code.

Reply:
Sure! making the changes.